src/stabilization_guide.md

```diff
@@ -0,0 +1,181 @@
+
+# Request for stabilization
+
+    Once an unstable feature has been well-tested with no outstanding
+    concern, anyone may push for its stabilization. It involves the
+    following steps.
+
+    - Documentation PRs
+    - Write a stabilization report
+    - FCP
+    - Stabilization PR
+
+## Documentation PRs
+
+    Prepare PRs to update documentations involing this new feature.
+    You need to submit PRs for  repositories The Reference, The Book
```

Comment:
It looks like this used to be a bulleted list?

Reply:
done